docs/v3.rst

```diff
@@ -0,0 +1,72 @@
+.. warning::
+
+    Do not use Connexion 3.0 in production yet!
+
+    Connexion 3.0 is currently available in alpha to give users the ability to test it and provide
+    feedback while we further polish the codebase and update the documentation.
```

Comment:
Small one: Let's make it explicit what the best way is for users to provide feedback? 
--> by creation an separate issue
--> as comment on a general feedback 3.x issue

Reply:
I'll add this in the announcement on Github. We can then later refer to that announcement in this doc, but currently it's a bit chicken and egg :)